spec/features/user_can_create_new_bill_spec.rb

```diff
@@ -2,11 +2,55 @@
 
 feature "user can add a new bill" do
   scenario "user successfully adds new bill" do
+    visit "/"
+    fill_in "datepicker", with: "2016-03-16"
+    fill_in "amount", with: "400"
+    fill_in "occupants", with: "5"
+    click_button "Create Bill"
+
+    expect(page).to have_content "You can save money on your energy bill"
+    expect(page).to have_css "svg#graph"
+    expect(page).to_not have_content "Date Bill is Due"
```

Comment:
I figured it would be an additional way to prove that the user was directed away from the new bill page. It isn't entirely necessary since the first `expect` covers that.